test/test_environment.rb

```diff
@@ -0,0 +1,13 @@
+require 'test/unit'
+require_relative '../lib/programr/environment'
+
+class TestEnvironment < Test::Unit::TestCase
+  def setup
+    @env = ProgramR::Environment.new
+  end
+
+  def test_readonly_tags_file
+    ProgramR::Environment.readonly_tags_file = 'test/data/readOnlyTags.yaml'
+    assert_equal(@env.get('bot_name'), 'test bot name')
+  end
```

Comment:
What do you think about adding a test to show that these tags are read-only as well, e.g. trying to assign to one and asserting it's the original value?